src/pages/LandingPage/Index.tsx

```diff
@@ -4,11 +4,13 @@ import Box from "@mui/material/Box";
 import NetworkInfo from "../Analytics/NetworkInfo/NetworkInfo";
 import UserTransactionsPreview from "./UserTransactionsPreview";
 import {AptosLearnBanner} from "./AptosLearnBanner";
+import {GithubDiscussionsBanner} from "./GithubDiscussionsBanner";
 
 export default function LandingPage() {
   return (
     <Box>
       <AptosLearnBanner />
+      <GithubDiscussionsBanner />
```

Comment:
Like the other one, should we hide the aptos learn banner now?

Reply:
Yes, lets please only have one banner at a time, unless there is a strong reason to have more

Reply:
removed